src/stylus/components/_tabs.styl

```diff
@@ -3,159 +3,161 @@
 
 /* Theme */
 tabs($material)
-  .tabs__li
+  background-color: $material.cards
+
+  .icon--left,
+  .icon--right
+    color: $material.icons.inactive
+
+  .tabs__div
     color: $material.tabs.active
 
     &.tabs__item--disabled
       color: $material.buttons.disabled
 
-  .tabs__bar
-    background-color: $material.cards
-
-    .icon--left,
-    .icon--right
-      color: $material.icons.inactive
-
-theme(tabs, "tabs__bar")
+theme(tabs, "tabs")
 
 .tabs
-  overflow: hidden
   position: relative
-  width: 100%
+  
+  .icon--prepend,
+  .icon--append
+    position: absolute
+    top: 0
+    width: 32px
+    align-items: center
+    cursor: pointer
+    user-select: none
 
-  &--grow .tabs__bar
-    .tabs__li
-      flex: 1
+  .icon--prepend
+    left: 4px
 
-  &--fixed > .tabs__bar
-    .tabs__li
-      min-width: 160px
-      max-width: 264px
-      flex: 0 1 160px
+  .icon--append
+    right: 4px
 
+  &--show-arrows
+    .tabs__wrapper
+      margin-left: 40px
+      margin-right: 40px
+
+  &--overflow      
+    .tabs__div
+      flex: 1 0 auto
+
+  &--align-with-title
+    .tabs__container > div:first-child,
+    .tabs__slider-wrapper + .tabs__div
+      margin-left: 56px
```

Comment:
These can just be padding on `.tabs__container`

Reply:
done